### DISCUSSION TITLE
Hook @AfterQueryMany

### DISCUSSION THREAD

**Discussion by DevBrasil** [TriPSs/nestjs-query#176] [Ideas]
Is there any way to handle the data after consulting the library, as I need to sign to a URL that is in the DB.

**Comment by TriPSs** [comment]
Multiple ways:
1. You can overwrite the [QueryService](https://tripss.github.io/nestjs-query/docs/persistence/typeorm/custom-service/)
2. You can add `@ResolveField` to your resolver to manage one field

**Comment by tonytoth** [comment]
I think using interceptors would be a better idea.
You can take a look here:
https://docs.nestjs.com/interceptors#aspect-interception
Why I think it is better ?
Because this will help you batch requests, deal with asynchronous better and you will have your logic decoupled from nestjs-query without making a mess.
